_posts/2021-02-15-changelog-141.markdown

```diff
@@ -0,0 +1,96 @@
+* Detect [E0116](https://doc.rust-lang.org/error-index.html#E0116) and [E0117](https://doc.rust-lang.org/error-index.html#E0117) compiler errors
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6716 -->
+* Support async function by create function intention (by [@Stzx])
+
+  {% include gif-img.html path="/assets/posts/changelog-141/6716" w="700px" %}
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6591 -->
+* Add inspection that checks non-existent crates in `Cargo.toml`.
+  Note, the corresponding feature is disabled by default for now.
+  To use it please enable `org.rust.crates.local.index` experimental feature (by [@Kobzol])
+
+## Performance Improvements
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6719 -->
+* Fix performance regression in name resolution for large modules
+
+## Fixes
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6768 -->
+* Improve `File | Open` action to set up project structure of Cargo project if a project is opened via directory with `Cargo.toml`
+
+* [New name resolution engine](https://github.com/intellij-rust/intellij-rust/issues/6217) improvements:
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6731 -->
+    * Enable new name resolution engine on the nightly plugin channel by default
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6733 -->
+    * Support resolve multi-segment macro paths
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6692 -->
+    * Fix completion of unqualified macro calls in some cases
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6723 -->
+    * Fix freeze after running tokio test
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6782 -->
+* Fix processing of `break`/`continue` inside block expression during control-flow analysis
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6804 -->
+* Fix enum rendering while debugging with GDB in CLion 2021.1
+
+* Improvements of `Change signature` refactoring (`Refactor | Change Signature` or <kbd>Ctrl</kbd>+<kbd>F6</kbd>):
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6795 -->
+    * Apply the refactoring to the called function when invoked on function and method calls (by [@Kobzol])
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6724 -->
+    * Fix auto-import in dialog when using new resolve
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6725 -->
+* Fix associated type bounds with super trait
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6688 -->
+* Fix false-positive [E0277](https://doc.rust-lang.org/error-index.html#E0277) in some cases
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6767 -->
+* Fix exception during loading structure of Cargo workspace with several IDE modules
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6805 -->
+* Add implicit build arguments to the beginning of the command line
```

Comment:
Let's rephase it to show user-visible changes instead implementation detail

Reply:
@mchernyavsky I mentioned it in https://github.com/intellij-rust/intellij-rust/pull/6805#issuecomment-775500566